app/scripts/migrations/092.2.test.ts

```diff
@@ -0,0 +1,92 @@
+import { NetworkType, toHex } from '@metamask/controller-utils';
+import { NetworkStatus } from '@metamask/network-controller';
+import { version as currentStateVersion, migrate } from './092.2';
+
+const TEST_NETWORK_CONTROLLER_STATE = {
+  networkId: 'network-id',
+  networkStatus: NetworkStatus.Available,
+  providerConfig: {
+    type: NetworkType.rpc,
+    chainId: toHex(42),
+    nickname: 'Funky Town Chain',
+    ticker: 'ETH',
+    id: 'test-network-client-id',
+  },
+  networkDetails: { EIPS: {} },
+  networkConfigurations: {
+    'network-configuration-id-1': {
+      chainId: toHex(42),
+      nickname: 'Localhost 8545',
+      rpcPrefs: {},
+      rpcUrl: 'http://localhost:8545',
+      ticker: 'ETH',
+    },
+  },
+};
+
+const anyPreviousStateVersion = 91;
+
+describe('migration #96', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should update the state version number in the appropriate metadata field', async () => {
+    const originalVersionedState = {
+      meta: { version: anyPreviousStateVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(originalVersionedState);
+
+    expect(newStorage.meta).toStrictEqual({ version: currentStateVersion });
+  });
+
+  it('should return state unaltered if there is no network controller state', async () => {
+    const originalMetaMaskState = {
+      anotherController: 'another-controller-state',
+    };
+    const originalVersionedState = {
+      meta: { version: anyPreviousStateVersion },
+      data: originalMetaMaskState,
+    };
+
+    const updatedVersionedState = await migrate(originalVersionedState);
```

Comment:
Nit: If we're comparing the inputs here to the outputs, we should deep clone them before passing them in. Otherwise the migration might still mutate the input, and the test wouldn't catch it.

In our case we deep clone the inputs anyway, but better for the test to catch that mistake if we ever make it.

e.g.
```suggestion
    const updatedVersionedState = await migrate(cloneDeep(originalVersionedState));
```

Reply:
I see that this was done for the first test, but the remaining tests still don't use it yet